releases/release-6.5.3.md

```diff
@@ -0,0 +1,159 @@
+---
+title: TiDB 6.5.3 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 6.5.3.
+---
+
+# TiDB 6.5.3 Release Notes
+
+Release date: xxx, 2023
+
+TiDB version: 6.5.3
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v6.5/production-deployment-using-tiup) | [Installation packages](https://www.pingcap.com/download/?version=v6.5.3#version-list)
+
+## Compatibility changes
+
+- note 1
+
+## Improvements
+
++ TiDB
+
+    <!--tw:ran-huang-->
+    - Improve the performance of `TRUNCATE` on partitioned tables with Placement Rules [#43070](https://github.com/pingcap/tidb/issues/43070) @[Lloyd-Pottiger](https://github.com/Lloyd-Pottiger)
+    - Avoid invalid Stale Read retries after resolving locks [#43659](https://github.com/pingcap/tidb/issues/43659) @[you06](https://github.com/you06)
+    - Reduce latency by using leader read when Stale Read encounters the `data-is-not-ready` error [#765](https://github.com/tikv/client-go/pull/765) @[Tema](https://github.com/Tema)
+    <!--tw:oreoxmt-->
+    - Add `Stale Read OPS` and `Stale Read MBps` metrics to track hit rate and traffic when using Stale Read [#43325](https://github.com/pingcap/tidb/issues/43325) @[you06](https://github.com/you06)
+
++ TiKV
+
+    <!--tw:ran-huang-->
+    - Reduce traffic by using gzip to compress `check_leader` requests [#14839](https://github.com/tikv/tikv/issues/14839) @[cfzjywxk](https://github.com/cfzjywxk)
+
++ PD
+
+    <!--tw:ran-huang-->
+    - Use a separate gRPC connection for PD leader election to prevent the impact of other requests [#6464](https://github.com/tikv/pd/pull/6454) @[rleungx](https://github.com/rleungx)
```

Comment:
```suggestion
    - Use a separate gRPC connection for PD leader election to prevent the impact of other requests [#6403](https://github.com/tikv/pd/issues/6403) @[rleungx](https://github.com/rleungx)
```